src/shared/components/login/login.css

```diff
@@ -9,12 +9,12 @@
 }
 
 .loginError {
-  color:red;
+  color: red;
   font-weight: bold;
 }
 
 select {
-  border: 1px solid #ccc;
+  border: 1px solid #cccccc;
```

Comment:
This isn't something new to this PR, but I just noticed this - won't this CSS style targeting `select` elements cause scoping issues? I know we've seen that in the past w/ CSS Modules, element styles, and global CSS scope.